Resources/Prototypes/SS220/Entities/Clothing/Uniforms/Blueshield_uniform.yml

```
@@ -0,0 +1,10 @@
+- type: entity
+  parent: UnsensoredClothingUniformBase
+  id: ClothingUniformJumpsuitBlueShield2
+  name: Униформа Синего Щита
+  description: Модная форма Синего Щита, как же она хороша.
```

```suggestion
  description: Модная форма Синего Щита - как же она хороша!
```
Кажется, так звучит лучше, что думаешь?